src/amr/messengers/hybrid_hybrid_messenger_strategy.hpp

```diff
@@ -34,6 +34,35 @@ namespace PHARE
 {
 namespace amr
 {
+
+    // this structure is a wrapper of a field*
+    // so to serve as a ResourcesUser for the ResourcesManager
+    template<typename FieldT>
+    struct FieldUser
+    {
+        struct Property
+        {
+            std::string name;
+            typename HybridQuantity::Scalar qty;
+        };
+        FieldUser(std::string fieldName, FieldT* ptr, typename HybridQuantity::Scalar qty)
+            : name{fieldName}
+            , f{ptr}
+            , quantity{qty}
+        {
+        }
+        std::string name;
+        FieldT* f;
+        typename HybridQuantity::Scalar quantity;
+        using field_type = FieldT;
+
+        std::vector<Property> getFieldNamesAndQuantities() const { return {{name, quantity}}; }
+
+        void setBuffer([[maybe_unused]] std::string const& bufferName, FieldT* field) { f = field; }
```

Comment:
```
void setBuffer(std::string const& /*bufferName*/, FieldT* field) { f = field; }
```

Reply:
curious : what's your reason for that VS maybe_unused? (in this case we know it's not maybe?)

Reply:
> in this case we know it's not maybe?

yeah and it's slightly easier on the eyes